JenkinsfileStageMerge

```diff
@@ -0,0 +1,87 @@
+// Openshift project
+openshiftProject = "continuous-infra"
+DOCKER_REPO_URL = '172.30.254.79:5000'
+STABLE_LABEL = "stable"
+
+// Defaults for SCM operations
+env.ghprbGhRepository = env.ghprbGhRepository ?: 'CentOS-PaaS-SIG/ci-pipeline'
+env.ghprbActualCommit = env.ghprbActualCommit ?: 'master'
+
+// Add new images here
+imageList = ["jenkins-continuous-infra-slave", "rpmbuild", "ostree-compose", "ostree-image-compose", "rsync", "singlehost-test", "ostree-boot-image", "linchpin-libvirt"]
+imageOperations = []
+
+library identifier: "ci-pipeline@${env.ghprbActualCommit}",
+        retriever: modernSCM([$class: 'GitSCMSource',
+                              remote: "https://github.com/${env.ghprbGhRepository}",
+                              traits: [[$class: 'jenkins.plugins.git.traits.BranchDiscoveryTrait'],
+                                       [$class: 'RefSpecsSCMSourceTrait',
+                                        templates: [[value: '+refs/heads/*:refs/remotes/@{remote}/*'],
+                                                    [value: '+refs/pull/*:refs/remotes/origin/pr/*']]]]])
+
+pipeline {
+    agent {
+        kubernetes {
+            cloud 'openshift'
+            label 'merge-trigger-' + env.ghprbActualCommit
+            containerTemplate {
+                name 'jnlp'
+                args '${computer.jnlpmac} ${computer.name}'
+                image DOCKER_REPO_URL + '/' + openshiftProject + '/jenkins-continuous-infra-slave:' + STABLE_LABEL
+                ttyEnabled false
+                command ''
+            }
+        }
+    }
+    stages {
+        stage("Detect Images to Promote") {
```

Comment:
There are actually no Dockerfiles maintained in this repo. They are all in ci-pipeline. So, pretty much all [merge] has to do is merge the PR. Which I think is line 70